modules/core/02-client/client/cli/tx.go

```diff
@@ -294,10 +294,6 @@ func NewCmdSubmitRecoverClientProposal() *cobra.Command {
 
 	flags.AddTxFlagsToCmd(cmd)
 	govcli.AddGovPropFlagsToCmd(cmd)
-	err := cmd.MarkFlagRequired(govcli.FlagTitle)
-	if err != nil {
```

Comment:
i think this was deprecated in govv1

Reply:
Sorry, just looking at this again. Where did you see that was not needed? I see that CLIs in SDK still do this ([here](https://github.com/cosmos/cosmos-sdk/blob/934abbcebb7b9ecb2ec9161bb26db18518d9580b/x/upgrade/client/cli/tx.go#L130) and [here](https://github.com/cosmos/cosmos-sdk/blob/934abbcebb7b9ecb2ec9161bb26db18518d9580b/x/upgrade/client/cli/tx.go#L182)).

Reply:
[the "title" ](https://github.com/cosmos/cosmos-sdk/blob/v0.47.5/x/gov/client/cli/tx.go)

Reply:
since we were targetting sdk 47.5 with this feature branch initially, but now i do see in [sdk 50](https://github.com/cosmos/cosmos-sdk/blob/v0.50.0-rc.0/x/gov/client/cli/tx.go#L23C18-L23C18) it seems to be undeprecated...?

Reply:
aha! @julienrbrt explains: `In v0.46 with the introduction to gov v1 these were deprecated. However in v0.47+ we've reintroduced title and summary in the proposal (instead of only in the metadata), hence the un-deprecation.`